README.md

```diff
@@ -175,4 +176,21 @@ Commit your code to the repository and deploy it to Heroku using Git.
 $ git add .
 $ git commit -am "make it better"
 $ git push heroku master
-```
\ No newline at end of file
+```
+
+## Mongo DB
+
+Create a new account within Mongo DB and select the version '''I'm learning MongoDB''' option at the initial sign up.  
+
+Initiate a local cluster with a free tier
+
+Create a new user and password - make a note of these credentials as these will be required later
+
+Also allocate your local IP address via the Network Access option
+
+poetry add pymongo 'pymongo[srv]'
+poetry run python 
+
+Created MongoBD.py 
+
+Within this file is where you will need to include your credentials to access your MondoDB cluster; please ensure that you protect your keys accordingly.  
```

Comment:
This sounds like you expect the variables to be set directly but your `mongodb.py` file actually expects them in environment variables - not sure if that's a change of strategy but it isn't quite how you've set it up (or at least, doesn't need to be!)